tests/test_order_lines.py

```diff
@@ -0,0 +1,44 @@
+from mollie.api.objects.order_line import OrderLine
+from mollie.api.objects.list import List
+
+ORDER_ID = 'ord_kEn1PlbGa'
+ORDER_LINE_ID = 'odl_dgtxyl'
+
+
+def test_get_order_lines(client, response):
+    """Retrieve lines of a single order by order ID"""
+    response.get('https://api.mollie.com/v2/orders/{order_id}'.format(order_id=ORDER_ID), 'order_single')
+
+    order = client.orders.get(ORDER_ID)
+    lines = order.order_lines
+    assert isinstance(lines, List)
+
+    for line in lines:
+        assert isinstance(line, OrderLine)
+        assert line.resource == 'orderline'
+
+    # Test properties of the first line
+    for line in lines:
```

Comment:
Don't use a loop when you don't intend to actually loop. Use `line = next(lines)` instead.